department-of-veterans-affairs/caseflow
# Confirm assignments modal

Connects #5822 

### Description
This PR adds the modal on the confirm assignments page. A future PR will add the functionality for sending the request to the backend.

### Testing Plan
1. Go to `hearings/schedule/build/upload`
1. Fill out the form and click `Continue`
1. On the following page, `Confirm assignments`
1. Confirm modal displays correctly
1. Confirm clicking `Go back` makes the modal disappear
1. Confirm clicking `Confirm upload` redirects you to the homepage
